addons/udes_stock/models/stock_picking_type.py

```diff
@@ -24,6 +22,8 @@ class StockPickingType(models.Model):
 
     show_operations = fields.Boolean(default=True)
 
+    goods_in = fields.Boolean()
```

Comment:
would be good to have help string as to what this flag means/does

Reply:
and ideally we would split black format commit with the [ADD] commit :)

Reply:
A few questions/points. :)

1. Why do we need this field? Goods In will be its own picking type.
2. We should put a string so that it is displayed like a title, the Odoo default would be set to `Goods in`, but `Goods In` looks more proper/professional.
3. As `stock.picking.type` is a core Odoo module, we should be prefixing new fields with `u_`.

Reply:
> and ideally we would split black format commit with the [ADD] commit :)

I will split the commits after I have fixed everything thanks.

Reply:
> A few questions/points. :)
> 
> 1. Why do we need this field? Goods In will be its own picking type.
> 2. We should put a string so that it is displayed like a title, the Odoo default would be set to `Goods in`, but `Goods In` looks more proper/professional.
> 3. As `stock.picking.type` is a core Odoo module, we should be prefixing new fields with `u_`.

1) Yes as per the discussion on my other MR I will most likey remove this, thanks.
2) If for some reason it stays in I will add one thanks.
3) OK I will try and do so from now on thanks.